.circleci/config.yml

```diff
@@ -37,7 +37,10 @@ jobs:
           command: |
             python -m venv venv
             . venv/bin/activate
-            pip install -e git+https://github.com/plotly/dash.git#egg=dash[testing]
+            git clone --depth 1 https://github.com/plotly/dash.git dash-main
+            cd dash-main && pip install -e .[dev,testing] --progress-bar off && cd ..
+            cd dash-main/\@plotly/dash-generator-test-component-nested && npm ci && npm run build && sudo R CMD INSTALL . && cd ../../..
```

Comment:
@alexcjohnson has simplified this in a previous PR - you can run `cd dash-main && npm run setup-tests && cd ..` instead for the test components

Reply:
Oh. Dev to master, no need to fix it. Just take note :)